CHANGELOG.md

```diff
@@ -24,8 +24,22 @@ docs/releases/Hotfix-Documentation-Best-Practices.md
 
 INTERNAL NOTE
 -->
+## Flutter 3.27 Changes
+
+### [3.27.0](https://github.com/flutter/flutter/releases/tag/3.27.0)
+Initial stable release.
+
 ## Flutter 3.24 Changes
 
+### [3.24.5](https://github.com/flutter/flutter/releases/tag/3.24.5)
+- [flutter/158125](https://github.com/flutter/flutter/pull/158125) - [iOS] Fixed a tool issue causing failures when `flutter build ios-framework --xcframework` copies Flutter debug symbols.
+- [flutter/56301](https://github.com/flutter/engine/pull/56301) - [Android] Fixes a crash on Android devices when the surface is released unexpectedly when using PlatformView's.
```

Comment:
It is probably more helpful to link to https://github.com/flutter/engine/pull/55277

Reply:
The already included pull request includes more information than the one you linked.  

We typically link the pull request or issue that matches the template.  Would you suggest replacing the link with the one you provided?

Reply:
Merging this in the current state, will update further next week if necessary.